wqflask/wqflask/metadata_edits.py

```diff
@@ -705,3 +727,26 @@ def approve_data(resource_id: str, file_name: str):
             "warning",
         )
     return redirect(url_for("metadata_edit.list_diffs"))
+
+def create_delta_csv(base_csv, form_data, sample_list):
+    base_csv_lines = base_csv.split("\n")
+    delta_csv_lines = [base_csv_lines[0]]
+
+    for line in base_csv_lines[1:]:
```

Comment:
Hi, maybe there's a way to code this that doesn't have such a high [cyclomatic](https://scribe.rip/@commbigo/python-code-complexity-metrics-e6c87646c1c2) complexity?

Otherwise #781 LGTM